frontend/src/pages/product-search/ProductSearch.jsx

```diff
@@ -203,14 +216,13 @@ function ProductSearch() {
                             />
                         ))}
                     </div>
+                    <br/>
+                    {productData.length > 0 && <button
+                        onClick={handleLoadMoreProducts}
+                        className="alternative-button primary-button-size center-button"
+                    >Load more</button>}
                 </div>
-
-                <br/>
-                {productData.length > 0 && <button
-                    onClick={handleLoadMoreProducts}
-                    className="alternative-button primary-button-size center-button"
-                >Load more</button>}
-                <Footer />
```

Comment:
Why no Footer on this page?

Reply:
Endless scroll (which btw, i think there's too many products shown so limiting it will help!)